app/weddingproject/settings.py

```diff
@@ -20,7 +21,7 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = ''
+SECRET_KEY = config('SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
maybe we can put:
DEBUG = config('DEBUG', default=True, cast=bool)
since DEBUG is for the devs.

This way we wouldn't have to touch this script and we could decouple all parameters by putting them in a different file.
Then, when we want to change some config, there is only one file to search for.
And maybe later we can also separate ALLOWED_HOSTS if necessary.

Reply:
It is a good idea. However it is out of the scope of this PR. You can create a task for it and add a description with what you think should be done.

Reply:
Good idea. I will check the other tasks and add a new one where it is more feasible.